Code/GraphMol/FilterCatalog/FilterMatchers.h

```diff
@@ -350,12 +356,10 @@ class RDKIT_FILTERCATALOG_EXPORT SmartsMatcher : public FilterMatcherBase {
   template <class Archive>
   void load(Archive &ar, const unsigned int version) {
     ar &boost::serialization::base_object<FilterMatcherBase>(*this);
-    {
-      RDUNUSED_PARAM(version);
-      std::string res;
-      ar &res;
-      d_pattern = boost::shared_ptr<ROMol>(new ROMol(res));
-    }
+    RDUNUSED_PARAM(version);
```

Comment:
Since we may use the version at some point in the future I prefer to leave the variable name in the header file so that it's clear what that argument is.